statsd/client/base.py

```diff
@@ -34,15 +44,21 @@ def timing(self, stat, delta, rate=1):
             delta = delta.total_seconds() * 1000.
         self._send_stat(stat, '%0.6f|ms' % delta, rate)
 
-    def incr(self, stat, count=1, rate=1):
+    def incr(self, stat: str, count: int = 1, rate: float = 1) -> None:
```

Comment:
I think this might accept non-integer values, at least, there's a test for that case https://github.com/jsocol/pystatsd/blob/aed950421ffeff606d8867311f59c2a32a12b3ef/statsd/tests.py#L152

```suggestion
    def incr(self, stat: str, count: float = 1, rate: float = 1) -> None:
```